base_bytecode_test.py

```diff
@@ -23,10 +23,10 @@ def setup_interp(self, filename, argc, argv):
         self.output = []
         module = interpreter.create_module(filename)
         global_state = State()
-        interpreter.load_globals(module, global_state, argc, argv)
-        functions = interpreter.load_func_table(module)
-        main_fun = LLVMGetNamedFunction(module, "main")
-        return (module, global_state, functions, main_fun)
+        module.load_globals(global_state, argc, argv)
```

Comment:
They can be retrieved from the module. No need to lookup functions inside the interpreter anymore, all that is inside the W_Module now.